test/VisionModel/resnet.jl

```diff
@@ -1,12 +1,10 @@
+using Flux
+using FluxModels
+using Test
+
 @testset "ResNet" begin
-  for MODEL in [
-    ResNet18,
-    ResNet34,
-    ResNet50,
-    ResNet101,
-    ResNet152,
-    ]   
-    model = MODEL()
-    @test size(model(rand(256, 256, 3, 50))) == (1000, 50)
+  for model in [ResNet18, ResNet34, ResNet50, ResNet101, ResNet152]
+    m = model()
+    @test size(m(rand([256, 256, 3, 50]))) == (1000, 50)
```

Comment:
The Float64 fallbacks for NNlib are really slow on CPU. Better to use Float32

```suggestion
    @test size(m(rand(Float32, 256, 256, 3, 50))) == (1000, 50)
```

Reply:
There is a method mismatch error arising. I guess I have to make some changes in ``src/VisionModel/resnet.jl``